src/ccl/charts/chart.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, ViewEncapsulation, Inject } from '@angular/core';
+import { ChartService } from '../services/chart.service';
+import { LineGraph } from './line-graph.component';
+
+/*
+ * Chart component
+ * Management for the sections that appear in the chart container
+ */
+
+@Component({
+  selector: 'charts',
+  encapsulation: ViewEncapsulation.None,
+  providers: [ChartService],
+  directives: [LineGraph],
+  styleUrls: [
+  '../../assets/css/chart.css',
+  ],
+  templateUrl: './chart.template.html'
+})
+export class Chart {
+    // Pull active chart data
+    public chartService: ChartService;
```

Comment:
From what I've seen, this is an unnecessary declaration, if in the constructor you do:
```
constructor(private chartService: ChartService) { }
```